scripts/masterCron.ts

```diff
@@ -105,11 +108,12 @@ async function runTX(
       continue;
     }
 
-    let gasPrice = await gas(network);
+    let newGasPrice = (await gas(network)).toString();
 
     try {
       const tx = await vault.connect(signer)[`${method}()`]({
-        gasPrice,
+        gasPrice: newGasPrice,
+        gasLimit: GAS_LIMIT,
```

Comment:
Btw the GAS_LIMIT only applies for `commit()`, it's larger for other TV functions

Reply:
ah yes, edited